### ISSUE TITLE
Google Detects usage as "suspicious activity" and signs out of the entire device.

### ISSUE THREAD

**Issue by j-romchain** [minecraft-linux/mcpelauncher-manifest#1021]
**Describe the bug**
When using this launcher, using manual sign in mode (I couldn't get built-in browser to work and find manual mode easier than using the built-in webview or whatever anyway) I have found that on each of the 2 days on which I have used it by the end of the day I had a security alert on my phone and was logged out of my google account on everything on my PC, and had to re-sign in everywhere. Am I doing something wrong?

**To Reproduce**
1. Install
3. Sign in using manual mode, using inspect to copy the oauth and user id cookie values, and putting in the email address associated with the account and setting a password.
4. Play Minecraft bedrock several times throughout the day
5. Close the launcher and wait up to 24 hours
6. Run the launcher again
7. you will likely find that the launcher reports no longer being signed in, and if you were logged in to chrome it is also signed out
8. upon signing back in to your google account in chrome if you were signed in you will be presented with a security warning about "suspicious activity" that was "helpfully" signed out for you.

**Expected behavior**
No security warnings at all, and at the very least not severe enough to cause automatic signing out of everything on the involved device.

**Screenshots**
![image](https://github.com/user-attachments/assets/20ec2535-09a5-41f2-80ef-248781b54298)

**Desktop (please complete the following information):**
 - OS: Linux
 - Version: mcpelauncher-manifest/jammy,now 0.15.2.9818777132.1~jammy amd64 [installed] (as per apt list) (along with mcpelauncher-ui-manifest)
 - Installation Type: Ubuntu Apt repository

**Additional context**
I wish to thank you for all you've done here to get this working.

**Comment by Illumye** [comment]
So I panicked for nothing this morning.... I installed the launcher on September 9th and it happened to me today....

**Comment by j-romchain** [comment]
Yep. I don't know how they would go about making Google not think it is suspicious. Perhaps make a mobile app that you install on your phone and get the apk that way? That way it can be relatively seamless for noobs and simpletons like myself still? Or perhaps it is functional if you sign in using its built in browser but triggers security when using an external browser because the user agents (the part of the browsers that tell the website what device type it is) don't line up?

**Comment by CullenJWebb** [comment]
This happened to my brother. So relieved to find that it wasn't a real vulnerability.

If this can't be patched then the GUI needs to warn users that this may happen.

**Comment by marpisco** [comment]
i had my computer turned off and received the "suspicious alert" notification.

uninstalling this bs and changing my google password
